components/interview/interfaces/name-generator/NameGenerator.tsx

```diff
@@ -80,7 +80,7 @@ function NameGenerator(_props: InterviewStage) {
     <div className={cn(interfaceWrapperClasses, 'flex grow flex-col gap-4')}>
       <Prompts prompts={demoPrompts} currentPromptId="1" />
       <div className="relative flex h-full min-h-0 flex-1 items-start justify-center gap-4">
-        <NodePanels panels={demoPanels} />
+        <NodePanels currentPromptId="1" panels={demoPanels} />
```

Comment:
We shouldn't need to do this, because prompts have their own ID which is unique to the stage - not the prompt.